### DISCUSSION TITLE
不太明白手机扫码连接电脑后怎么使用

### DISCUSSION THREAD

**Discussion by handbye** [wanghongenpin/proxypin#86] [Q&A] [answered]
是手机上的流量可以转发到电脑上吗？

**Comment by wanghongenpin** [comment]
对，省了配置WiFi步骤，手机端app得大开着，还有就是安卓部分请求 WiFi配置代理抓不到，比如flutter请求，但是扫码可以抓到

**Reply by handbye** [reply]
我使用ios扫码后，发现wifi代理那里并没有改，而且手机上的流量也没有转发到macos上。

**Reply by wanghongenpin** [reply, marked as answer]
手机端得开着抓包，因为手机端用得VPN，不用wifi代理。vpn会拦截流量

**Reply by handbye** [reply]
感谢，这次确实抓到包了，但是ios扫码后会自动关闭抓包，这个感觉得改一下，感谢提供这么好的工具！
